lib/utilities/pristine.js

```diff
@@ -85,7 +86,9 @@ function installPristineApp(appName, options) {
 
   chdir(temp.pristinePath);
 
-  var promise = applyCommand('new', appName, extraOptions)
+  var args = extraOptions.concat(['-sb', '-sn', runCommandOptions]);
```

Comment:
Not sure why you add the '-sb' and '-sn' options unconditionally although there is `extraOptions` taking care of them?